src/modules/voxin.h

```diff
@@ -0,0 +1,60 @@
+/*
+  libvoxin API, v 1.4.6 copied from
+  https://raw.githubusercontent.com/Oralux/libvoxin/2ad36d8f7c07ea80663c9b3524de443abdf76758/src/api/voxin.h
+
+  Copyright: 2016-2020 Gilles Casse <gcasse@oralux.org>
+  License: LGPL-2.1+ or MIT
+ 
+*/
+#ifndef VOXIN_H
+#define VOXIN_H
+
+#include <stdint.h>
+#include "eci.h"
+
+#define LIBVOXIN_VERSION_MAJOR 1
+#define LIBVOXIN_VERSION_MINOR 4
+#define LIBVOXIN_VERSION_PATCH 6
+
+typedef enum {voxFemale, voxMale} voxGender;
+typedef enum {voxAdult, voxChild, voxSenior} voxAge;
+
+#define VOX_ECI_VOICES 22
+#define VOX_RESERVED_VOICES 30
+#define VOX_MAX_NB_OF_LANGUAGES (VOX_ECI_VOICES + VOX_RESERVED_VOICES)
+#define VOX_STR_MAX 128
```

Comment:
Such kind of hardcoded sizes always pose problem: they either

* are too big and thus eat memory for no use
* are too small and thus truncates names etc.

Better use pointers to dynamically-allocated strings than inlining arrays in the structure.